lib/src/executables/boilerplate_upgrade.dart

```diff
@@ -80,7 +80,7 @@ void main(List<String> args) {
     <Suggestor>[
       PropsAndStateClassesSimpleMigrator(),
       AdvancedPropsAndStateClassMigrator(),
-      StubbedPropsAndStateClassRemover(),
+      PropsAndStateCompanionClassRemover(),
```

Comment:
We'd want to run this conditionally based on whether or not the props / state class was updated, wouldn't we?